opencart 2.2.0.0/upload/catalog/controller/smailyforopencart/cron_customers.php

```diff
@@ -72,6 +73,9 @@ public function index() {
                     die('Error with request to Smaily API, try again later.');
                 }
             }
+            $settings['smaily_for_opencart_sync_time'] = date('c');
+            $this->model_smailyforopencart_helper->editSetting('smaily_for_opencart', $settings);
```

Comment:
I think you should implement updating the last synchronization time with `editSettingValue` method, because current implementation could override other module settings with older values, as customer sync can take a long time to run, and user can change module settings during that time.